jquery.flot.js

```diff
@@ -1459,14 +1497,12 @@ Licensed under the MIT license.
                 switch(type) {
                     case 'min':
                     case 'max':
-                        // this is a workaround to display the endpoints with a higher
-                        //precision without changing the API and all the tickFormatters
-                        //axis.tickDecimals = (axis.tickDecimals != null ? axis.tickDecimals + 1 : null);
-                        label = axis.tickFormatter(v, axis);
-                        //axis.tickDecimals = (axis.tickDecimals != null ? axis.tickDecimals - 1 : null);
+                        // display endpoints with higer precision
+                        var precision = plot.computeValuePrecision(axis.min, axis.max, axis.direction, axis.options);
```

Comment:
ahh, I see. But I think we should respect the user settings even even for the end labels. I might be wrong but I think desktop does that either. 

For normal tick the precision is computed using a delta of the entire interval shown deivided by the number of ticks.

For the end labels I think the precision should be computed taking an interval one pixel wide around the end label (the value of that in axis units can be calculated using axis.p2c functions). In other words the label should show the most round number in that small region.

Let me know what you think about this algorithm and if it makes sense.